File: android/src/main/kotlin/com/baseflow/permissionhandler/data/PermissionGroup.kt
package com.baseflow.permissionhandler.data

import com.google.gson.annotations.SerializedName

enum class PermissionGroup {
    @SerializedName("unknown")
    UNKNOWN,
    @SerializedName("calendar")
    CALENDAR,
    @SerializedName("camera")
    CAMERA,
    @SerializedName("contacts")
    CONTACTS,
    @SerializedName("location")
    LOCATION,
    @SerializedName("microphone")
    MICROPHONE,
    @SerializedName("phone")
    PHONE,
    @SerializedName("photos")
    PHOTOS,
    @SerializedName("reminders")
    REMINDERS,
    @SerializedName("sensors")
    SENSORS,
    @SerializedName("sms")
    SMS,
    @SerializedName("storage")
    STORAGE,
    @SerializedName("speech")
    SPEECH,
    @SerializedName("locationAlways")
    LOCATION_ALWAYS,
    @SerializedName("locationWhenInUse")
    LOCATION_WHEN_IN_USE,
    @SerializedName("mediaLibrary")
    MEDIA_LIBRARY
}

File: ios/Classes/data/PermissionGroup.swift
//
//  PermissionGroup.swift
//  permission_handler
//
//  Created by Maurits van Beusekom on 25/07/2018.
//

import Foundation

enum PermissionGroup : String, Codable {
    case calendar = "calendar"
    case camera = "camera"
    case contacts = "contacts"
    case location = "location"
    case locationAlways = "locationAlways"
    case locationWhenInUse = "locationWhenInUse"
    case mediaLibrary = "mediaLibrary"
    case microphone = "microphone"
    case photos = "photos"
    case reminders = "reminders"
    case sensors = "sensors"
    case speech = "speech"
}
